src/extensions/common-api/index.ts

```diff
@@ -19,27 +19,21 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-// Lens-extensions api developer's kit
-export { LensMainExtension } from "../lens-main-extension";
-export { LensRendererExtension } from "../lens-renderer-extension";
-
 // APIs
 import * as App from "./app";
 import * as EventBus from "./event-bus";
 import * as Store from "./stores";
 import * as Util from "./utils";
-import * as Interface from "../interfaces";
+import * as Registrations from "./registrations";
```

Comment:
This does not sounds good as top-level module. Would it make sense to have these as `Types`?

Reply:
Oh because they are not usually used by y themselves? Yeah in `Types` makes sense.